relay-general/src/protocol/security_report.rs

```diff
@@ -1523,7 +1523,11 @@ mod tests {
         let mut event = Event::default();
         Csp::apply_to_event(json.as_bytes(), &mut event).unwrap();
         let message = &event.logentry.value().unwrap().formatted;
-        insta::assert_debug_snapshot!(message, @r###""Blocked \'image\' from \'google.com\'""###);
+        insta::assert_debug_snapshot!(message, @r###"
```

Comment:
Could you snapshot these as `message.as_str()` instead?

Reply:
Replaced with `as_str().unwrap()`, as `message` is `Annotated`